styles.css

```diff
@@ -140,6 +142,16 @@ p {
   padding: 10px;
 }
 
+/*
+  At this point the contact links start to overlap with the content on the page.
+  Since this info is reduntant from the contact page anyway, let's just hide it.
+*/
+@media screen and (max-width: 1100px) {

```

Comment:
Same thing as above, we can better reorganize the content on the page by using a layout method and avoid needing to outright hide important info like social links. We should also add the social links to the footer.

We can also have the text replaced by the icon as the screen gets smaller.